### ISSUE TITLE
Incorrect behavior of // prettier-ignore

### ISSUE THREAD

**Issue by OnkelTem** [prettier/plugin-php#1643]
Look how the `static` keyword disappearing. 

**@prettier/plugin-php v0.16.1**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14BMLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCADsAOgCMIFjAht4slQGF0AW19sAAoAEoH4roKGIBxuAMZQAMowAJ5q1g4wHLgs0ACOgvDQgW7KAXpyCiDEIhAwxWylbOUGAL4sUBTwMOgoaPBYkqBs2DDNIm1VmGCCo3ggADwA-MZgADqY65hsvlDYYGzxAAQAImzsALLE8RCBsoLK1aurAGKIshTYANzr68JQJ2c2JdMNcfNAngBRAAe8T6WCet3uxGhsIwmG+mD+AIuVyQYKgCLuDw2q0RxNeUHeACFhlAMVjTjiQXiegTSUTiISkZgKe8AJIkJAyeIY9YAegAVBL1ocJYcAAKQdBgJApQ5k4i8igCnQQYX-bDoQQQeIyuWKiDK1XJQ4jCAqADaAF1DmoAG6U7BmsXrWKyYbYQ40-3Mw5QKE6TAfdUcnlvCg0kSHYAyw5KnSxHQUW3sBCxQ4AEjAsb52YAvIdVtE5GxmVWMYdDmKxWnoHBEBAALTwZSYVmpkZneD5qSCEFow7KWAJWLUKABLzUDwUaCYAAUAEpk6nG23jZg02xVzA8HgpzAZ3OF0v3qvN4cANSHB07xuHADkhuN8XfhzLAD5nw-AB9FIVV-ACP32Wp4Dkd8mBAvICmgCgIMA0QACUAFUIQQ99gOghA4L-QD3ykeNfydJhX0bd93U9NCgPwsCoEY99020ZJwLw4CkMKDpGMwnCeI4ljGJfA83yk5iuNYkiPx0CN4MQqB8n41D5KE3CaMOJ0nVfJ0G0OFp1hMzAQBaIA)

**Input:**
```php
<?php

namespace DataMincerPlugins\Fields;

use DataMincerCore\Exception\PluginException;
use DataMincerCore\Plugin\PluginFieldBase;
use DataMincerCore\Plugin\PluginFieldInterface;

/**
 * @property PluginFieldInterface source
 * @property string[] levels
 */
class Balance extends PluginFieldBase {
  protected static $pluginId = "balance";
  // prettier-ignore
  static function getSchemaChildren() {
    return parent::getSchemaChildren() + [
      'source' => [ '_type' => 'partial', '_required' => TRUE, '_partial' => 'field' ],
      'levels' => [ '_type' => 'prototype', '_required' => TRUE, '_prototype' => [
        '_type' => 'text', '_required' => TRUE,
      ]]
    ];
  }
}

```

**Output:**
```php
<?php

namespace DataMincerPlugins\Fields;

use DataMincerCore\Exception\PluginException;
use DataMincerCore\Plugin\PluginFieldBase;
use DataMincerCore\Plugin\PluginFieldInterface;

/**
 * @property PluginFieldInterface source
 * @property string[] levels
 */
class Balance extends PluginFieldBase {
  protected static $pluginId = "balance";
  // prettier-ignore
  function getSchemaChildren() {
    return parent::getSchemaChildren() + [
      'source' => [ '_type' => 'partial', '_required' => TRUE, '_partial' => 'field' ],
      'levels' => [ '_type' => 'prototype', '_required' => TRUE, '_prototype' => [
        '_type' => 'text', '_required' => TRUE,
      ]]
    ];
  }
}

```

**Comment by czosel** [comment]
Thanks for the bug report @OnkelTem !

**Comment by OnkelTem** [comment]
Hehe, thanks for the most convenient way to report issues — the button in the right bottom in the playground!

**Comment by czosel** [comment]
Should be fixed in `v0.17.6` :tada:
